.github/workflows/checks.yaml

```diff
@@ -71,14 +71,10 @@ jobs:
       - name: target
         run: "rustc -vV | sed -n 's|host: ||p'"
         if: matrix.os != 'windows-2022'
-      - name: Install cargo-llvm-cov
-        uses: taiki-e/install-action@56ab7930c591507f833cbaed864d201386d518a8
-        with:
-          tool: cargo-llvm-cov
-      - name: Install cargo-nextest
-        uses: taiki-e/install-action@56ab7930c591507f833cbaed864d201386d518a8
+      - name: Install cargo-llvm-cov & cargo-nextest
+        uses: taiki-e/install-action@a48a50298f98c47e46a957ae6f82c44cc4878e42
         with:
-          tool: cargo-nextest
+          tool: cargo-llvm-cov,cargo-nextest
```

Comment:
this now uses one invocation of the install action (which I've updated) to install `cargo llvm-cov` and `cargo nextest`